scio-core/src/main/scala/com/spotify/scio/coders/instances/kryo/SchemaSerializer.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.coders.instances.kryo
+
+import com.esotericsoftware.kryo.Kryo
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.twitter.chill.KSerializer
+import org.apache.avro.Schema
+
+private[coders] class SchemaSerializer extends KSerializer[Schema] {
+  private lazy val parser = new Schema.Parser()
```

Comment:
I think we can mark this as `@transient` so it's not encoded as part of a `Coder[Schema]`